From 545a4b4d22cf0ed01fdb3a737e456efa5af246f9 Mon Sep 17 00:00:00 2001
From: Kenneth Graunke <kenneth@whitecape.org>
Date: Thu, 20 May 2021 11:25:35 -0700
Subject: [PATCH 03/20] iris: Delete a comment suggesting we use tiled staging
 buffers

We basically tried this, and it performed worse, so delete the
suggestion in the comments that we may want to do it someday.

Acked-by: Lionel Landwerlin <lionel.g.landwerlin@intel.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/10941>

diff --git a/src/gallium/drivers/iris/iris_resource.c b/src/gallium/drivers/iris/iris_resource.c
index c6050d84590..3b36af1b7d9 100644
--- a/src/gallium/drivers/iris/iris_resource.c
+++ b/src/gallium/drivers/iris/iris_resource.c
@@ -1989,8 +1989,7 @@ iris_transfer_map(struct pipe_context *ctx,
     *
     * Images are less clear-cut.  Resolves can be destructive, removing some
     * of the underlying compression, so we'd rather blit the data to a linear
-    * temporary and map that, to avoid the resolve.  (It might be better to
-    * a tiled temporary and use the tiled_memcpy paths...)
+    * temporary and map that, to avoid the resolve.
     */
    if (!(usage & PIPE_MAP_DISCARD_RANGE) &&
        !iris_has_invalid_primary(res, level, 1, box->z, box->depth)) {
-- 
2.36.1

